### PR TITLE
lesson 9 claswork exampls

### PR DESCRIPTION
**Pull request by MrTim**
Classwork for lesson 9. Just a simple speed calculation.

### CODE REVIEW

File: converter/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        function calculateSpeed(distance, time) {
+            return distance/time;
+        }
+
+        function getUserInput() {
+            const distance = document.getElementById('distance').value;
+            const time = document.getElementById('time').value;
+
+            return {
+                distance, time
```
Comment (caritosteph) [inline converter/index.html]: I am not sure if they going to get return this format. 
probably is better if it is assign to a variable and then return the variable.

**Review by caritosteph — APPROVED** [review]
I just left one comment, but the rest looks fine.

File: converter/index.html (hunk 1 above)
Comment (MrTim) [reply converter/index.html]: I fixed it. But students should not see this code, they should implement it by themselfs :)

Comment (caritosteph) [reply converter/index.html]: oh you are right. 😅

**Pull request merged**